util/QueryHelper.js

```diff
@@ -0,0 +1,40 @@
+class QueryHelper {
```

Comment:
So, hum. how about using this as a middleware instead ? You prepare the Model and avoid to do this splitting into each controller

Reply:
So, start the QueryBuilder and attach it to req.locals and then the handlers use that existing QueryBuilder to add the where clause(s)?